index.html

```diff
@@ -0,0 +1,125 @@
+                    <span class="material-icons">close</span>
+                </a>
+
+                <a href="./index.html" onclick="closeMenu()">
+                    Home
+                </a>
+
+                <a href="./about.html" onclick="closeMenu()">
+                    About
+                </a>
+            </div>
+        </div>
+    </header>
+    <main>
+        <section class="introduction">
+            <h4>console.log("Hello World!")</h4>
+            <h1>BEST CODING BOOTCAMP, CodeFire</h1>
+            <span>Join the best coding bootcamp, winer of multiple awards and forming a lot of current professional developers since 2016.</span>
+            <h3>Be part of the next tech revolution with our help</h3>
+            <h5>Go to contact page for more info</h5>
+        </section>
+        <section id="program-description">
+            <h3 id="program-header" class="sfont">How is life in CodeFire </h3>
+                <div class="descriptions">
+                    <div class="program-content">
+                        <span class="material-icons">schedule</span>
+                        <h3 class="card-title sfont">Self driven</h3>
+                        <h5 class="card-description sfont">Complete the program at your own pace, only you and your will</h5>
+                    </div>
+
+                    <div class="program-content">
+                        <span class="material-icons">support_agent</span>
+                        <h3 class="card-title sfont">Support</h3>
+                        <h5 class="card-description sfont">Get support from our staff members whenever you need it!</h5>
+                    </div>
+
+                    <div class="program-content">
+                        <span class="material-icons">face</span>
+                        <h3 class="card-title sfont">Social</h3>
+                        <h5 class="card-description sfont">Improve your social skills by talking to other students!</h5>
+                    </div>
+
+                    <div class="program-content">
+                        <span class="material-icons">code</span>
+                        <h3 class="card-title sfont">Resources</h3>
+                        <h5 class="card-description sfont">Learn from our top-tier coding resources</h5>
+                    </div>
+
+                    <div class="program-content">
+                        <span class="material-icons">group_add</span>
+                        <h3 class="card-title sfont">Friendly</h3>
+                        <h5 class="card-description sfont">Overall we have a friendly enviroment where everyone is free to share anything!</h5>
+                    </div>
+
+                    
+                </div>
+                
+                <button type="button" id="join-btn" class="sfond">Join CodeFire</button>
+                
+        </section>
+
+        <section id="speakers">
+            <h3 id="speakersHeader">Featured speakers</h3>
```

Comment:
Provide a space between your image and text:
![Screen Shot 2021-09-08 at 19 27 14](https://user-images.githubusercontent.com/22258274/132602954-ef8f64bf-1422-4d2d-b5b4-a5a43040adc3.png)

Reply:
* Fixed 👍